internal/thelma/charts/repo/index/index.go

```diff
@@ -60,7 +60,7 @@ func (index *index) HasVersion(chartName string, version string) bool {
 func (index *index) Versions(chartName string) []string {
 	var versions []string
 
-	if index.Entries == nil || len(index.Entries) == 0 {
```

Comment:
In recent versions of go, the builtin map type has been updated such that taken the len() of a nil map is now 0